src/NServiceBus.RabbitMQ.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.26430.4
```

Comment:
Ideally I'd prefer not to see this change in a hotfix but, since it's only tooling, I think it's OK to let it in this time.